contributors/guide/README.md

```diff
@@ -168,9 +168,17 @@ _Improvements needed_
 
 ## Testing
 
-_Improvements needed_
-* link to testing process 
-* walkthrough of where to find what in the tests (how to use, how to debug)
+Testing is the responsibility of all contributors and is in part owned by all sigs, but is also coordinated by [sig-testing](https://github.com/kubernetes/community/tree/master/sig-testing).
+
+The main testing overview document is [here](https://github.com/kubernetes/community/blob/master/contributors/devel/testing.md).
+
+There are three coarse types of test in kubernetes.  The location of the test code varies with type, as does the specifics of the environment needed to successfully run the test:
```

Comment:
s/coarse/core/ ?

Reply:
just dropped the word leaving "three types"